lib/signaling/v2/room.js

```diff
@@ -52,6 +52,13 @@ class RoomV2 extends RoomSignaling {
     }, options);
 
     localParticipant.setBandwidthProfile(options.bandwidthProfile);
+
+    const { options: { signaling_region: signalingRegion, audio_processors: audioProcessors = [] } } = initialState;
+    localParticipant.setSignalingRegion(signalingRegion);
+
+    audioProcessors.push('rnnoise');
```

Comment:
Are we going to ship this feature with rnnoise by default?

Reply:
No, this is internal use only.